packages/odf/components/odf-dashboard/object-storage-card/capacity-card.tsx

```diff
@@ -21,7 +21,7 @@ const ObjectCapacityCard: React.FC = () => {
     endpoint: 'api/v1/query' as any,
   });
 
-  const dataFrames = !loaded && !error ? parseMetricData(data) : [];
+  const dataFrames = loaded && !error ? parseMetricData(data) : [];
```

Comment:
```suggestion
  const dataFrames = !loaded && !error ? parseMetricData(data) : [];
```
Earlier it was correct, it is bad naming convention. It should be renamed to `loading` because it is Prometheus polling response...